scripts/pack_server.sh

```diff
@@ -109,8 +109,28 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+if [[ $separate_servers == "false" ]] && [[ ! -f ${BUILD_LATEST_DIR}/output/bin/pegasus_server/pegasus_server ]]; then
+    echo "ERROR: ${BUILD_LATEST_DIR}/output/bin/pegasus_server/pegasus_server not found"
+    exit 1
+fi
+
+if [[ $separate_servers == "true" ]] && [[ ! -f ${BUILD_LATEST_DIR}/output/bin/pegasus_meta_server/pegasus_meta_server ]]; then
+    echo "ERROR: ${BUILD_LATEST_DIR}/output/bin/pegasus_meta_server/pegasus_meta_server not found"
+    exit 1
+fi
+
+if [[ $separate_servers == "true" ]] && [[ ! -f ${BUILD_LATEST_DIR}/output/bin/pegasus_replica_server/pegasus_replica_server ]]; then
+    echo "ERROR: ${BUILD_LATEST_DIR}/output/bin/pegasus_replica_server/pegasus_replica_server not found"
+    exit 1
+fi
```

Comment:
Will `copy_file` throw an error if the source file not exist? You can remove the manual judgements if it will.

Reply:
> Will `copy_file` throw an error if the source file not exist? You can remove the manual judgements if it will.

`copu_file` will throw an error if the source file not exist, i already remove those manual judgements.